pulsar-transformations/src/test/java/com/datastax/oss/pulsar/functions/transforms/TransformFunctionTest.java

```diff
@@ -126,6 +139,90 @@ void testDropFields() throws Exception {
     assertNull(valueAvroRecord.getSchema().getField("valueField3"));
   }
 
+  @Test
+  void testMatchingPredicate() throws Exception {
+    String userConfig =
+        (""
+            + "{\"steps\": ["
+            + "    {\"type\": \"drop-fields\", \"fields\": \"keyField1\", \"when\": \"${key.keyField1 == 'key1'}\"},"
```

Comment:
I think that the user should not write ${ and }. We can add them automatically

Reply:
Sure. on that note, should we mention in out public docs that the we use expression language behind the scenes? 

We need to either do that (so user know which syntax to use inside the ${}) OR document supported syntax ourselves (which is a subset of UEL)